services/web-app/styles/vendor/_overrides.scss

```diff
@@ -5,13 +5,37 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+@use '@carbon/styles/scss/utilities/z-index' as z-index;
```

Comment:
Josh pointed out to me that as of the most recent version of `@carbon/react`, we can pull this right from `@carbon/react/scss/utilities/z-index` to keep all of our sass uses coming exclusively from `@carbon/react`. I think we have this one and potentially one other usage of the zindex from styles in the web app that could be udpated.

Reply:
I created a separate issue for this. Outside the scope of this PRs intent. https://github.com/carbon-design-system/carbon-platform/issues/431

Reply:
👍 We can toss this into next sprint.